src/prefect/artifacts.py

```diff
@@ -0,0 +1,160 @@
+from typing import Union
+
+from prefect import context, Client
+
+
+def running_with_backend() -> bool:
```

Comment:
Would this be useful / better placed elsewhere?

Reply:
Yeah I can make an artifacts utility file

Reply:
From talking to @cicdw there was an intention to stop the utilities from creeping larger. I was curious if it fit into any of the existing utility groups or modules.

Reply:
To be super clear, the utility comment I mentioned previously was because it was a proposed new utility to DRY out a single line of code, which is not a good tradeoff.

Reply:
My 2c on things like these is to keep the helper functions local to the module they're used until there's a good reason for another module to need it. Since they're non-public, moving helpers around doesn't harm anything. But adding things to a utility module adds indirection (I have to go look at another file to understand some code), and can harden interfaces too early (can I change this helper function? what if something else relies on its current signature?).